src/advanced/node-management.md

```diff
@@ -406,18 +409,18 @@ all messages in the event queue are processed.
 
 #### Parameters
 
-None.
+null
```

Comment:
I'm not sure I can agree with this change. *Logically,* there are no parameters; passing `null` to the server is a convention.